src/language.js

```diff
@@ -350,16 +339,16 @@ const pages = {
     title: 'Management Regimes Overview',
     noDataMainText: 'This project has no submitted sample units yet.',
     noDataSubText:
-      'This page will show the management regime of submitted sample units by method and site.',
+      'This page will show the Management Regime of submitted sample units by method and site.',
   },
   copySiteTable: {
     title: 'Copy Sites',
-    filterToolbarText: 'Filter this table by name, project or country',
+    filterToolbarText: 'Filter this table by name, project, or country',
     copyButtonText: 'Copy selected sites to project',
   },
   copyManagementRegimeTable: {
     title: 'Copy Management Regimes',
-    filterToolbarText: 'Filter management regimes by name, project, or year',
+    filterToolbarText: 'Filter Management Regimes by name, project, or year',
```

Comment:
On seeing this in context, I think we should keep this lowercased. To be consistent we would have to capitalize the entities in `Filter this table by name, reef type, reef zone, or exposure` and I think that's going overboard. But I would leave the other MR capitalization as you have it.